packages/repository/src/decorators/model.ts

```diff
@@ -16,10 +21,36 @@ export const PROPERTY_KEY = 'loopback:property';
  * @param definition
  * @returns {(target:any)}
  */
-export function model(definition?: Object) {
+export function model(definition?: {
+  name: string;
+  properties?: {[name: string]: PropertyDefinition | PropertyType};
+  settings?: {[name: string]: any};
+}) {
```

Comment:
Should we define an interface for this arg?

Reply:
I like that idea, it should make the code cleaner.

Reply:
Done in 33a087b